pom.xml

```diff
@@ -69,10 +69,11 @@
         <jahia-module-type>system</jahia-module-type>
         <jahia-deploy-on-site>system</jahia-deploy-on-site>
         <jahia-module-signature>MCwCFAmbHhYWLvU2QLiip3YdaVlENPEOAhRFWugTeoZqba9GJhPhOfaSaqIJmw==</jahia-module-signature>
+        <jahia.plugin.version>6.9</jahia.plugin.version>
         <export-package>org.jahia.modules.apitokens</export-package>
         <import-package>
-            graphql.annotations.annotationTypes;version="[7.2,9)",
-            graphql.annotations.connection;version="[7.2,8)",
+            graphql.annotations.annotationTypes;version="[7.2,99)",
+            graphql.annotations.connection;version="[7.2,99)",
             graphql.schema;version="[13.0,14)",
```

Comment:
No need to do this here graphql.schema;version="[13.0,22)", ? like in https://github.com/Jahia/server-availability-manager/pull/93/files

Reply:
yes, sorry, I forgot to push the last commit :-p